localizely.yml

```diff
@@ -0,0 +1,9 @@
+config_version: 1.0
+project_id: ead34689-ec52-41d9-b675-09bc85a6cbd7
+file_type: flutter_arb
+upload:
+  files:
+    - file: mobile/assets/i18n/en-US.json
+      locale_code: en
+    - file: mobile/assets/i18n/de-DE.json
```

Comment:
Should this be in the upload section? I believe we should have just en-US in the upload section, then have all the other languages in the download section so localizely updates them for us automatically?

Reply:
Personally I would add all existing JSON files to the upload section. That way manual JSON changes and localizely are always in-sync. Localizely seems to do a great job merging local and remote changes.